src/support/d.ts

```diff
@@ -133,7 +133,7 @@ function findByKey(
 	target.children
 		.forEach((child, index) => {
 			if (isNode(child)) {
-				if (found) {
+				if (found && found.found) {
```

Comment:
do you think we could name `found` better? like perhaps `nodeInfo`?

Reply:
Yes that makes more sense. Updated